vividus-plugin-browserstack/src/test/java/org/vividus/selenium/browserstack/BrowserStackCapabilitiesConfigurerTests.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.selenium.browserstack;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.vividus.bdd.context.IBddRunContext;
+import org.vividus.bdd.model.RunningStory;
+
+@ExtendWith(MockitoExtension.class)
+class BrowserStackCapabilitiesConfigurerTests
+{
+    @Mock private IBddRunContext bddRunContext;
+    @Mock private DesiredCapabilities desiredCapabilities;
+    @InjectMocks private BrowserStackCapabilitiesConfigurer configurer;
+
+    @AfterEach
+    void afterEach()
+    {
+        verifyNoMoreInteractions(bddRunContext, desiredCapabilities);
+    }
+
+    @Test
+    void shouldNotConfigureCapabilitiesIfBrowserStackInDisabled()
+    {
+        configurer.setBrowserStackEnabled(false);
+        configurer.configure(desiredCapabilities);
```

Comment:
this test doesn't look correct: it has neither `assert` not `verify`
(I see the logic of `afterEach`, but that's wrong: single test logic is spread across multiple methods without explicit relation)

Reply:
hmm why single test logic is spread? afterEach() contains test logic that is applicable for all the tests in this class, additionally the verification in afterEach makes all verifications in this test redundant

Reply:
https://github.com/vividus-framework/vividus/pull/1067 unit testing agreement added